style.css

```diff
@@ -174,3 +174,81 @@ hr {
 .lang:hover {
   color: #172b4d;
 }
+
+.contact {
+  background-image: url(images/contact\ form\ background.png);
```

Comment:
- Hey, please, let's ensure we've used the correct background in the contact section.